pulseapi/entries/views.py

```diff
@@ -214,14 +214,15 @@ def post(self, request, *args, **kwargs):
             ids = self.request.query_params.get('ids', None)
             queryset = Entry.objects.public()
 
-            if ids is not None:
+            def parse_int(id):
                 try:
-                    ids = [int(x) for x in ids.split(',')]
-                except:
-                    return Response(
-                        "Invalid entry id",
-                        status=status.HTTP_400_BAD_REQUEST
-                    )
+                    return int(id)
+                except ValueError:
+                    return id
+
+            def remove_str(id):
+                return isinstance(id, int)
+
 
             def bookmark_entry(id):
```

Comment:
we should be able to just fix up this function by swapping `except Entry.DoesNotExist:` for just `except:`